Editor/Resources/EditorWindow/common.uss

```diff
@@ -285,6 +285,7 @@ Image#ExternalLinkIcon {
     border-color: var(--color-text-input-border);
     background-color: var(--color-text-input-background);
     font-size: var(--font-size-large);
+    flex: 1 1 auto;
```

Comment:
Is this needed for all text inputs or is it only for this popup? If it's only this popup, add it to the file specific css